CONTRIBUTING.md

```diff
@@ -86,6 +86,18 @@ expanding into more specifics.
    with a few days. Although, depending on the circumstances, it may take
    longer.
 
+### New sources, sinks, and transforms
+
+If you're contributing a new source, sink, or transform to Vector, thank you that's way cool! There's a couple of things you need think about if you want to make sure we can merge your contribution.
```

Comment:
It might be worth adding a note that we're often able to help out with these things or do them in followup PRs, but they are a blocker for changes to be fully released.